benchmarks/generate_data_footer_benchmark.py

```
@@ -0,0 +1,570 @@
+#
+# dec_frequency_opr
+#
+
+
+def gen_grid_freq(n_rows, n_cols, default_val, gen_func):
+    default_probability = 0.95
+    data = [[default_val for _ in range(n_cols)] for _ in range(n_rows)]
+
+    for i in range(n_rows):
+        for j in range(n_cols):
+            if not random.random() < default_probability:
```

Ruff (0.11.9)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)